scripts/CustomMaterials.zs

```diff
@@ -9,46 +9,93 @@ import mods.gregtech.recipe.Utils;
 
 //MATERIALS 8000-10000 RESERVED FOR NUCLEAR-RELATED ITEMS
 //MATERIALS 10000-20000 RESERVED FOR FLUIDS
-//MATERIALS 20000-30000 RESERVED FOR SOLIDS
-//MATERIALS 29000-30000 RESERVED FOR ELEMENTS
+//MATERIALS 20000-29000 RESERVED FOR SOLIDS
+//MATERIALS 29000-30000 RESERVED FOR ORES
+//MATERIALS 30000-31000 RESERVED FOR ELEMENTS
 //MATERIALS 31000-32727 RESERVED FOR ISOTOPES
 
 //----------------------------------ELEMENTS---------------------------------
 
-MaterialBuilder(29000, "iodine")
+MaterialBuilder(30000, "iodine")
 .element("Iodine")
 .dust()
 .color(0x390b6e)
 .build();
 
-MaterialBuilder(29001, "hafnium")
+MaterialBuilder(30001, "hafnium")
 .element("Hafnium")
 .dust()
 .ingot()
 .color(0x1c1b18)
 .build();
 
-MaterialBuilder(29002, "chromium")
+MaterialBuilder(30002, "chromium")
 .element("Chromium")
 .dust()
 .ingot()
 .color(0xccb8c7)
 .build();
 
-MaterialBuilder(29003, "strontium")
+MaterialBuilder(30003, "strontium")
 .element("Strontium")
 .dust()
 .ingot()
 .color(0xdbdaab)
 .build();
```

Comment:
All of these materials already exist in CEu. You just need to add dust/ingot/fluid or any other properties to make them visible and usable.